packages/example-corvair/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "example-corvair",
+  "version": "0.0.1",
+  "description": "An hello world example for react-server",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/redfin/example-react-server.git"
+  },
+  "keywords": [
+    "react-server",
+    "Isopmorphic",
+    "Universal",
+    "JavaScript",
+    "React"
+  ],
+  "author": "",
+  "license": "MIT",
```

Comment:
We're going apache license on the other projects. Probably makes sense on this one too?